src/types.ts

```diff
@@ -138,6 +146,8 @@ export interface ApiFastWithdrawal extends ApiStarkwareSigned {
 
 export interface ApiKeyResponseObject {
   apiKey: string;
```

Comment:
Did we change this on our backend to `key`, so `ApiKeyResponseObject` and `ApiKeyCredentials` are now the same?

Reply:
We did - let me update this